rtl/eclrtl/rtldynfield.cpp

```diff
@@ -30,67 +30,83 @@
 
 //---------------------------------------------------------------------------------------------------------------------
 
-const RtlTypeInfo *FieldTypeInfoStruct::createRtlTypeInfo() const
+const RtlTypeInfo *FieldTypeInfoStruct::createRtlTypeInfo(bool useBias) const
```

Comment:
not sure about useBias as the name of the parameter.
- unsigned fields aren't biased
- string fields aren't translated.
- fields in the payload aren't biased.
- there are some weird rules for integers inside keyed rows - I have a feeling they don't work.

isLegacyKeyedField may be better